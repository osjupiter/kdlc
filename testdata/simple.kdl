scene "SimpleScene" {
    node "Button" x=100 y=100 width=200 height=50 {
        component "Button" text="Click me"
    }
}