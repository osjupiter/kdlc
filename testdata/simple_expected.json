{
  "scene": {
    "arg1": "SimpleScene",
    "node": {
      "arg1": "Button",
      "height": 50,
      "width": 200,
      "x": 100,
      "y": 100,
      "component": {
        "arg1": "Button",
        "text": "Click me"
      }
    }
  }
}